functions.sh

```diff
@@ -180,7 +180,7 @@ make_dist(){
     target_tar_gz=${target_tar}.gz
 
     tar_gz_file=${target_folder}.tar.gz
-    tar_gz_timestamp_file=${target_folder}-${TIMESTAMP}.tar.gz
+    tar_gz_timestamp_file=${target_folder}-${TIMESTAMP}-${PYTHON_PACKAGE_VERSION}.tar.gz
```

Comment:
I think that we should remove the timestamp .... as the timestamp is arbitrary and does not help with versioning. Thanks!